docs/qs-backup-setup.md

```diff
@@ -82,81 +89,155 @@ You should obtain a layout similar to this one (it is fine if you only have one
 
-<img src="/content/images/backup-setup/bitshelter-restore-previous-version.png" data-origin="content/images/backup-setup/bitshelter-restore-previous-version.png" alt="">
+2. Click on the copy you'd like to go back to:
+
+<img src="/content/images/backup-setup/bitshelter-restoremenu.png" data-origin="content/images/backup-setup/gitsetup-bitshelter-restoremenu" alt="">
+
+3. You can then click restore, to overwrite current version of the folder, or click open to browse the files:
+
+<img src="/content/images/backup-setup/bitshelter-restoremenu2.png" data-origin="content/images/backup-setup/gitsetup-bitshelter-restoremenu2" alt="">
+
+!> If you have issues with the above instuctions, you can also refer to [this guide](https://www.howtogeek.com/howto/11130/restore-previous-versions-of-files-in-every-edition-of-windows-7/) <img src="/content/images/backup-setup/bitshelter-restore-previous-version.png" data-origin="content/images/backup-setup/bitshelter-restore-previous-version.png" alt="">
 
 ### Internet backups: Git & Github
 
 #### Git 101
 
 ?> **Git** is a tool that makes it easier to track changes to files. When you edit a file, git can help you determine exactly *what* changed, *who* changed it, and *why*. <br /><br />It is useful for coordinating work among multiple people on a project, and for tracking progress over time by saving “checkpoints”. You could use it while writing an essay, or to track changes to artwork and design files. [\[1\]](https://hackernoon.com/understanding-git-fcffd87c15a3)
 
-For our purpose, **git** will be our mean to:
+**For our purposes**, ***git*** will be our mean to:
 - Create versions of our collection (*\"checkpoints\"*).
 - Upload\* our work to a safe\*\*, remote place.
 - *(Bonus)* Synchronize our work between devices.
 
-\*: GitHub has a 1GB limit on free private repositories.
-\*\*: *Although GitHub can be considered fairly reliable, we highly recommend that you implement a solution to encrypt your collection. This is, after all, a way to your most inner thoughts.*
+\*: GitHub has a 1GB limit on free private repositories. [GitLab](https://about.gitlab.com) offers 10gb free storage by default and [BitBucket](http://bitbucket.org) offers a 5gb plan for 3$. 
+
+\*\*: *Although GitHub can be considered fairly reliable, we highly recommend that you implement a solution to encrypt your collection. This is, after all, a way to your innermost thoughts.*
 
 #### Installing & Setting up Git with GitHub
 
-1. Create a [GitHub account](https://github.com/join/)
-2. [Follow this guide](https://vladmihalcea.com/tutorials/git/windows-git-ssh-authentication-to-github/) to:
-  - Install Git
-  - Setup a SSH key that automatically authenticate your Git installation with your GitHub account
+!> If you want to use SSH authentication instead of HTTPS, follow [this guide](https://vladmihalcea.com/tutorials/git/windows-git-ssh-authentication-to-github/)
```

Comment:
Same. Warnings should really only be used to catch the user's attention on important information, otherwise it loses its meaning.